### PR TITLE
Changelog for bower 1.7.0

### PR DESCRIPTION
**Pull request by prometheansacrifice**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
 
+## 1.7.0 - 2015-12-05
+- Updates `fs-write-stream-atomic`
+- Bower search shows a help display when no package name is specified. Fixes [#2066](https://github.com/bower/bower/issues/2066)
+- Updates only those packages that are explicitly requested by the user
+  Related Issues
+  - [#256](https://github.com/bower/bower/issues/256)
+  - [#924](https://github.com/bower/bower/issues/924)
+  - [#1770](https://github.com/bower/bower/issues/1770)
+- Implements [JSCS](http://jscs.info/overview.html) to enforce bower style guide ([#2043](https://github.com/bower/bower/issues/2043)
```
Comment (sheerun) [inline CHANGELOG.md]: I'll remove it since this change is not relevant for endusers

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+- Adds `bower --save functionality` ([#2035](https://github.com/bower/bower/issues/2035))
+- Fixes fetch command when used with svn ([#1650](https://github.com/bower/bower/issues/1650))
```
Comment (sheerun) [inline CHANGELOG.md]: It should rather say "Allow for @ in username for SVN on windows"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+## 1.7.0 - 2015-12-05
+- Updates `fs-write-stream-atomic`
+- Bower search shows a help display when no package name is specified. Fixes [#2066](https://github.com/bower/bower/issues/2066)
+- Updates only those packages that are explicitly requested by the user
+  Related Issues
+  - [#256](https://github.com/bower/bower/issues/256)
+  - [#924](https://github.com/bower/bower/issues/924)
+  - [#1770](https://github.com/bower/bower/issues/1770)
+- Implements [JSCS](http://jscs.info/overview.html) to enforce bower style guide ([#2043](https://github.com/bower/bower/issues/2043)
+- Adds `bower --save functionality` ([#2035](https://github.com/bower/bower/issues/2035))
```
Comment (sheerun) [inline CHANGELOG.md]: ``bower update --save` functionality`

File: CHANGELOG.md (hunk 1 above)
Comment (prometheansacrifice) [inline CHANGELOG.md]: What about bower search behaviour?

**Comment by sheerun** [comment]
We'll also need to add "Update bower-config" and copy-paste changes from there

File: CHANGELOG.md (hunk 1 above)
Comment (sheerun) [inline CHANGELOG.md]: JSCS is not relevant. `bower search` is :)

Comment (prometheansacrifice) [inline CHANGELOG.md]: Alright! Retaining it.

Comment (prometheansacrifice) [inline CHANGELOG.md]: And the new package update behaviour?

**Comment by prometheansacrifice** [comment]
@sheerun Add the following?

- Properly restore env variables if they are undefined at the beginning
- Properly handle default setting for config.ca
- Display proper error if .bowerrc is a directory instead of file

**Comment by sheerun** [comment]
This were changes in previous version of bower-config. We need to release new bower-config and update bower to use it.

**Comment by prometheansacrifice** [comment]
Right. Then it should be `Allow the use of environment variables in .bowerrc`. Can you confirm @sheerun There was no commit referring to the bump to `1.2.3`

**Comment by sheerun** [comment]
Yes

**Comment by sheerun** [comment]
And other things we did in bower-config since 1.2.3 release

**Comment by prometheansacrifice** [comment]
Closing as this was on the `master`

**Comment by sheerun** [comment]
Hm?

**Comment by prometheansacrifice** [comment]
Reopened the PR [here](https://github.com/bower/bower/pull/2083)

**Pull request closed without merging**